src/gam/gapi/cloudidentity/userinvitations.py

```diff
@@ -14,8 +14,8 @@
 def _get_customerid():
     ''' returns customer in format needed for this API'''
     customer = GC_Values[GC_CUSTOMER_ID]
-    if customer.startswith('C'):
-        customer = customer[1:]
+#    if customer.startswith('C'):
+#        customer = customer[1:]
```

Comment:
just take these out. Think I was testing something at one point and missed removing them because customer normally = my_customer.